client/components/telecom/telephony/abbreviatedNumbers/telecom-telephony-abbreviated-numbers.modal.js

```diff
@@ -1,34 +1,34 @@
 angular.module("managerApp").controller("telecomTelephonyAbbreviatedNumbersModal",
-                                        function ($scope, $q, $translate, $uibModalInstance, data, Toast) {
+                                        function ($scope, $q, $translate, $uibModalInstance, data) {
                                             "use strict";
 
                                             var self = this;
 
-                                            this.loading = {
+                                            self.loading = {
                                                 updating: false
                                             };
 
-                                            angular.extend(this, data);
+                                            angular.extend(self, data);
 
-                                            this.numberPattern = /^00\d{2,3}[\s\d]+$/;
-                                            this.namePattern = /^[a-zA-Z0-9\s]*$/;
+                                            self.numberPattern = /^00\d{2,3}[\s\d]+$/;
+                                            self.namePattern = /^[a-zA-Z0-9\s]*$/;
 
-                                            this.cancel = function () {
+                                            self.cancel = function () {
                                                 $uibModalInstance.dismiss("cancel");
                                             };
 
-                                            this.send = function () {
-                                                this.loading.updating = true;
-                                                $q.when(this.saveCallback({ value: this.data })).then(
+                                            self.send = function () {
+                                                self.errorMessage = null;
+                                                self.loading.updating = true;
+                                                $q.when(self.saveCallback({ value: self.data })).then(
                                                     function () {
                                                         $uibModalInstance.close(self.data);
                                                     },
                                                     function (err) {
                                                         if (/^This abbreviated/.test(err.data.message)) {
-                                                            $scope.abbreviatedNumberForm.abbreviatedNumber.$setValidity("available", false);
-                                                            Toast.error($translate.instant("telephony_abbreviated_numbers_not_free_error"));
```

Comment:
I removed toast usage because I'm not able to display messages in the modal, same problem in production.